hello-strongbox-gradle/build.gradle

```diff
@@ -9,8 +13,35 @@ compileJava {
 
 description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
 
-ext.username = System.getProperty('credentials.username') != null ? System.getProperty('credentials.username') : mavenUser
-ext.password = System.getProperty('credentials.password') != null ? System.getProperty('credentials.password') : mavenPassword
+// publishing to GitHub Package Registry
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/j4ckofalltrades/strongbox-examples")
+            credentials {
+                username = project.findProperty("gpr.user") ?: System.getenv("GITHUB_USERNAME")
+                password = project.findProperty("gpr.key") ?: System.getenv("GITHUB_TOKEN")
+            }
+        }
+    }
+    publications {
+        gpr(MavenPublication) {
+            pom {
+                description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
+            }
+
+            groupId = "org.carlspring.strongbox.examples"
+            artifactId = "hello-strongbox-gradle"
+            version = "1.1-SNAPSHOT"
```

Comment:
This can remain as `1.0-SNAPSHOT`. We don't intend to really ever bump the versions of the examples.

Reply:
Yeah my bad, I was testing something earlier -- i'll update this.